packages/jwt-service/src/types/IJwtService.ts

```diff
@@ -126,6 +127,7 @@ export interface JwsJsonGeneralWithIdentifiers extends JwsJsonGeneral {
 
 export interface JwsJsonSignatureWithIdentifier extends JwsJsonSignature {
     identifier: ExternalIdentifierResult
+    publicKeyHex: string
```

Comment:
not sure, but will this change break any existing implementations?

Reply:
It's a result interface, I do not think and extra field will break anything and if the interface was used anywhere else I would have had a compiler error in one of the libs using it by now.